.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
> What was the motivation for this change? Did you have problems with bundler?

I opened this PR after having red tests in Travis but green in local. The first thing I try when experiencing these issues is to have as much consistent environments as possible in order to reproduce the failures. When I checked I saw very different bundler versions.

The failirues in Travis turned out to be related to bundler. Somehow the `cancan` gem wasn't present in the `bundle/cache` folder which bundler uses in Travis to install the dependencies.

My point is that having the same version would reduce the number of possible culprits when this happens. I can't tell however whether the issue was directly related to the old bundler version.

In any case, I think is also good to have our dependencies up to date, in this case bundler.